taxcalc/functions.py

```diff
@@ -878,26 +873,33 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517, e37717,
     # imputation for x62730
     if e62730 != 0 and e24515 > 0:
         x62730 = e62730 - e24515
-    c62730 = e24515 + x62730
+    c62730 = e24515
 
     _amtded = c60200 + c60220 + c60240
-    _prefded = c60200 + c60220 + c60240
+    if FDED == 2:
```

Comment:
`FDED == 2` implies the code for standard deduction. But not sure if we need this if we were doing the optimization.